ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ObjectDiffusion/Inbound/V2/Decision.hs

```diff
@@ -0,0 +1,345 @@
+{-# LANGUAGE BlockArguments #-}
```

Comment:
I would avoid enabling this extension if it's only being used in L280 and L284.

Reply:
Sounds fair.